code/modules/events/event_dynamic.dm

```diff
@@ -64,9 +64,9 @@ var/list/event_last_fired = list()
 
 	possibleEvents[/datum/event/radiation_storm] = 50
 	if(active_with_role["Medical"] > 1)
-		//possibleEvents[/datum/event/viral_infection] = 30
+		possibleEvents[/datum/event/viral_infection] = 40
```

Comment:
touches event weights, needs cl

Reply:
of course. The final changelog will cover in priority every thing that the average player might encounter.